lib/ansible/plugins/action/win_reboot.py

```diff
@@ -81,8 +81,20 @@ def run(self, tmp=None, task_vars=None):
 
         result = super(ActionModule, self).run(tmp, task_vars)
 
-        # initiate reboot
-        (rc, stdout, stderr) = self._connection.exec_command("shutdown /r /t %d" % pre_reboot_delay_sec)
+        # Initiate reboot
+        (rc, stdout, stderr) = self._connection.exec_command('shutdown /r /t %d' % pre_reboot_delay_sec)
+
+        # Test for "A system shutdown has already been scheduled. (1190)" and handle it gracefully
+        if rc !=0 and '(1190)' in stderr:
```

Comment:
Ah, I wasn't sure that that error code was also the RC.
In Linux is it limited to 255 error codes :-)